src/org/usfirst/frc/team2175/loop/SmartDashboardLoop.java

```diff
@@ -52,9 +53,10 @@ protected void showGearIntakeInfo() {
     }
 
     private void populateAutonSelector() {
-        autonSelector.addDefault("Drive Forward and place gear on peg",
-                new DriveForwardAndPlaceGearOnPegAutonomous());
-        autonSelector.addObject("Turn Left Auton", new TurnLeftAutonomous());
+        // autonSelector.addDefault("Drive Forward and place gear on peg",
+        // new DriveInchesWithPercentVbusCommand(12));
+        autonSelector.addDefault("Default",
+                new DriveInchesWithArcadeDriveForAutonAutonomous());
```

Comment:
@NoahConnors We need to make sure we have good autonomous routines here. Not just "default".